src/main/java/io/github/mfoo/libyear/LibYearMojo.java

```diff
@@ -150,6 +155,19 @@ public class LibYearMojo extends AbstractMojo {
     @Parameter(property = "maxLibYears", defaultValue = "0.0")
     private float maxLibYears;
 
+    /**
+     * Path to the report file, if empty no report file will be generated.
+     */
+    @Parameter(property = "reportFile")
+    private String reportFile;
+
+    /**
+     * Whether the dependency should be included in the report. If it is set to "0", all dependencies will be included,
+     * otherwise only dependencies older than the specified number of years will be included.
+     */
+    @Parameter(property = "minLibYearsForReport", defaultValue = "0.0")
+    private float minLibYearsForReport;
```

Comment:
Would it be possible to add a test for this setting?

Reply:
The test "reportFileGenerated" is also the test for minLibYearsForReport. It is checked that not all outdated dependencies end up in the report. Should I write another separate test? What exactly should the test case be?